### PR TITLE
Add cask "wormhole"

### PR DESCRIPTION
**Pull request by singularitti**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/wormhole.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  version :latest
+  sha256 :no_check
+
+  # files.octopusgame.com was verified as official when first introduced to the cask
+  url 'https://files.octopusgame.com/os/WormholeInstaller.dmg'
+  name 'Wormhole'
+  homepage 'https://er.run/'
+
+  app 'Wormhole.app'
+end
```
Comment (ran-dall) [inline Casks/wormhole.rb:11]:
```suggestion

  zap trash: '~/Library/Saved Application State/er.Wormhole.savedState'
end
```

**Review by ran-dall — CHANGES_REQUESTED** [review]
@singularitti You're missing the `zap` stanza

File: Casks/wormhole.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'wormhole' do
+  version :latest
+  sha256 :no_check
+
+  # files.octopusgame.com was verified as official when first introduced to the cask
+  url 'https://files.octopusgame.com/os/WormholeInstaller.dmg'
+  name 'Wormhole'
```
Comment (ran-dall) [inline Casks/wormhole.rb:7]:
```suggestion
  appcast 'https://er.run/release'
  name 'Wormhole'
```

**Review by ran-dall — CHANGES_REQUESTED** [review]
@singularitti You're also missing the `appcast`.

File: Casks/wormhole.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'wormhole' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/wormhole.rb:2]: Needs a `version` (`1.1.9`) since an `appcast` will be set, as well as a `sha256`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by ran-dall** [comment]
```shell
$ /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/developer/bin/cask-pr-local-check https://github.com/Homebrew/homebrew-cask/pull/83842
==> Downloading https://files.octopusgame.com/os/WormholeInstaller.dmg
Already downloaded: /Users/draque/Library/Caches/Homebrew/downloads/4c66b28ea2d90baa785df05873c30ea94fb400024a25b7f44d4fd84206407422--WormholeInstaller.dmg
==> Verifying SHA-256 checksum for Cask 'wormhole'.
audit for wormhole: passed

1 file inspected, no offenses detected
```

**Review by ran-dall — APPROVED** [review]

**Pull request merged**